### DISCUSSION TITLE
Configurar postgis para QFieldCloud

### DISCUSSION THREAD

**Discussion by henryquiros** [opengisch/QField#4019] [Q&A]
Buenas Tardes 

Tengo una duda de como debo configurar el QFieldCloud para que realice los cambios en tiempo real, a la hora de modificar datos en QFIELD y que se vean reflejado en mi QGIS.
Agradezco cualquier colaboracion.

**Comment by ZsanettMed** [comment]
Hi, 
even if Spanish is a very nice language we would love if you could write in English. I used google-translator to understand your question, you can use it too. :) 

 (I have a question about how I should configure the QFieldCloud so that it makes the changes in real time, when modifying data in QFIELD and that they are reflected in my QGIS.
I appreciate any collaboration.)

The answer is: with the live PostGIS connection, you can find more information here: https://docs.qfield.org/get-started/tutorials/advanced-setup-qfc/#example-workflow-postgis

Best, 
Zsanett
